js/leaflet-idw.js

```diff
@@ -0,0 +1,354 @@
+/*
+ (c) 2016, Manuel B�r (www.geonet.ch)
```

Comment:
I think this is probably fine as long as the author is properly credited. However, if we can find a hosted version, that might be preferable.